.github/workflows/action.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - uses: julia-actions/setup-julia@latest
         with:
           version: ${{ matrix.julia-version }}
+      - run: julia --project -e 'using Pkg; Pkg.add(url="https://github.com/exanauts/BlockPowerFlow.jl", rev="master")'
```

Comment:
We might want to use a consistent workflow with a `deps/deps.jl` script to pull in the required dependencies. Let me know whether I should push that to this branch.